InvenTree/users/models.py

```diff
@@ -377,14 +393,22 @@ def get_permission_object(permission_string):
 
         (app, perm) = permission_string.split('.')
 
-        (permission_name, model) = perm.split('_')
+        # split up the permissions -> handle models with underscores
+        permission_name, *model = perm.split('_')
+        # handle models that have
+        if len(model) > 1:
+            app += '_' + '_'.join(model[:-1])
+            perm = permission_name + '_' + model[-1:][0]
+        model = model[-1:][0]
 
         try:
             content_type = ContentType.objects.get(app_label=app, model=model)
             permission = Permission.objects.get(content_type=content_type, codename=perm)
         except ContentType.DoesNotExist:
             logger.warning(f"Error: Could not find permission matching '{permission_string}'")
             permission = None
+        except Exception as _e:
+            print(_e)
```

Comment:
probably should throw `_e` into a `logger.error` call

Reply:
this is removed in the next commit; only needed it for debug